hibernate-core/src/test/java/org/hibernate/orm/test/orphan/onetomany/DeleteOneToManyOrphansTest.java

```diff
@@ -155,4 +157,20 @@ public void testReplacedWhileManaged(SessionFactoryScope scope) {
 		);
 	}
 
+	@Test
+	@TestForIssue(jiraKey = "HHH-15258")
```

Comment:
Since IDENTITY is required to trigger this (according to Jira), you'll also need to skip this test for databases which do not support IDENTITY. 

```suggestion
	@TestForIssue(jiraKey = "HHH-15258")
	@RequiresDialectFeature( DialectFeatureChecks.SupportsIdentityColumns.class )
```